pkg/asset/installconfig/gcp/validation_test.go

```diff
@@ -144,6 +156,24 @@ func TestGCPInstallConfigValidation(t *testing.T) {
 			expectedError:  true,
 			expectedErrMsg: "platform.gcp.project: Invalid value: \"invalid-project\": invalid project ID",
 		},
+		{
+			name:           "Valid public zone id",
+			edits:          editFunctions{addPublicZoneID},
+			expectedError:  false,
+			expectedErrMsg: "",
+		},
+		{
+			name:           "Invalid public zone id",
+			edits:          editFunctions{addPublicZoneID, invalidatePublicZoneID},
+			expectedError:  true,
+			expectedErrMsg: "invalid zone id",
```

Comment:
nit: the error message check should be more specific like https://github.com/openshift/installer/pull/2764/commits/52b42f85c0285441b661ef236855cd533f9771c4#diff-4ff55f77efed526a0a93d7ecf949b249R157

Reply:
Fixed!